apps/stable_diffusion/src/utils/stable_args.py

```diff
@@ -270,6 +270,20 @@ def path_expand(s):
     help="flag to clear all mlir and vmfb from common locations. Recompiling will take several minutes",
 )
 
+p.add_argument(
+    "--save_metadata_to_json",
+    default=False,
```

Comment:
Don't we want this to be true ? Or if this is false we should toggle the png metadata to true.

Reply:
I am honestly not sure which is preferred here. As far as I can tell the `app` and `web` versions are already writing metadata to a csv? I have been having trouble getting both GUI versions to run consistently on my machine (somehow 32GB does not seem like enough ram and I keep getting memory errors).

Reply:
For clarification, my initial intention and use case for these flags was for the CLI version of the program in `shark/examples/shark_inference/stable_diffusion`. That version does not write to .csv and defaults to a json file output.

Personally I like the idea of writing metadata to the png itself by default. It keeps the data with the image and the extra space cost is maybe only 1-2KB and is far cleaner in file browsers.

Reply:
Yeah I like it to be in the meta data too. I'm testing the new flow and the imports. Once that is good we can land your change and iterate on the right defaults.